deploy/addons/kubevirt/README.md

```diff
@@ -0,0 +1,45 @@
+## kubevirt Addon
```

Comment:
thank you for adding this readme, could u please this read me to our site section ?
maybe under tutorials ?

Reply:
Hey, I added the readme under the tutorials section.  Would you like the readme in the deploy  folder to be removed, or is it okay for it to live in both places?

Reply:
thank you we can have a readme in the addon folder but with a link to the website, so anyone goes to that can find out where is the tutorial for this addon.